.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+    "arrowParens": "avoid",
+    "bracketSameLine": false,
+    "bracketSpacing": true,
+    "printWidth": 100,
+    "quoteProps": "as-needed",
+    "semi": true,
+    "singleQuote": true,
+    "tabWidth": 4,
+    "trailingComma": "none"
+}
```

Comment:
Let's use tabs instead of spaces and in this case `tabWidth` becomes obsolete
Do we really need: `bracketSameLine`  and `bracketSpacing` in this project?

Reply:
Sounds good. Copy pasted this from my own project, I'll clean it up.